persistence/account_shared_sql.go

```diff
@@ -99,3 +101,17 @@ func (p *PostgresContext) insertAccount(accountType types.ProtocolAccountSchema,
 	}
 	return nil
 }
+
+func (p *PostgresContext) getParamsOrFlagsAtHeightQuery(tableName string, height int64, descending bool) string {
```

Comment:
I think `persistence/gov.go` would be the appropriate place for this function as well.